src/ts/components/subtitlesettings/subtitlesettingspanel.ts

```diff
@@ -33,34 +37,36 @@ export class SubtitleSettingsPanel extends SettingsPanel {
     this.overlay = config.overlay;
 
     let manager = new SubtitleSettingsManager();
+    // Quick lang getters
+    const i18fonts: FontPanelLabels = i18n.q.settings.fonts;
 
     this.config = this.mergeConfig(config, {
       components: <Component<ComponentConfig>[]>[
-        new SettingsPanelItem('Font size', new FontSizeSelectBox({
+        new SettingsPanelItem(i18fonts.fontSize, new FontSizeSelectBox({
```

Comment:
I would prefer if `FontPanelLabels` wasn't a separate interface but defined inline in the `Translation` interface so it purposefully cannot be used like this here and the strings would always have to be specified with their "fully qualified name", e.g. `i18n.q.settings.fonts.fontSize` instead of `i18fonts.fontSize`. It's more verbose, true, but better resembles the "usual" pattern of translation frameworks and might be more intuitive to understand.